### PR TITLE
Add Andalusian API and generate docs

### PR DESCRIPTION
**Pull request by miguelgcasado**

### CODE REVIEW

File: compmusic/dunya/andalusian.py [hunk 1]
```diff
@@ -0,0 +1,334 @@
...
+        {uuid, name, transliterated_name, display_order}
+
+    For additional information about each raag use :func:`get_mizan`.
+    """
+    return compmusic.dunya.conn._get_paged_json("api/andalusian/mizan")
+
+def get_mizan(mid):
+    """ Get specific information about a mizan.
+
+    :param rid: A mizan id or uuid
```
[the remark this answers is not in the document]
Comment (miguelgcasado) [reply compmusic/dunya/andalusian.py]: That's fine. I will use just the uuids. The only thing is that for the instruments there is no uuid, so I guess at this point you were using ids. Should we leave it like that?

[the remark this answers is not in the document]
Comment (alastair) [reply compmusic/dunya/andalusian.py]: It's possible that musicbrainz didn't support instrument mbids when we first wrote this importer. We should modify the schema to support mbids and import them from musicbrainz.

File: compmusic/dunya/andalusian.py [hunk 2]
```diff
@@ -0,0 +1,334 @@
...
+    :param amid: An artist id or uuid
+    :param mid: A mizan id or uuid
+    :param tid: A tab id or uuid
+    :param nid: An nawba id or uuid
+    :param fid: A form id or uuid
+    :param fbid: A instrument mbid
+    :param ibid: An artist mbid
+    :returns: A list of dictionaries containing work/s
+    """
+    path = 'work?artist={0}&mizan={1}&tab={2}&nawba={3}&form={4}&instrument={5}'
```
[the remark this answers is not in the document]
Comment (miguelgcasado) [reply compmusic/dunya/andalusian.py]: I think it makes sense to delete this function, we don't need it actually

File: compmusic/dunya/andalusian.py [hunk 3]
```diff
@@ -0,0 +1,334 @@
...
+    return compmusic.dunya.conn._get_paged_json("api/andalusian/artist", 
+            extra_headers=extra_headers)
+
+
+def get_artist(ambid):
+    """ Get specific information about an artist.
+
+    :param ambid: An artist mbid
+
+    :returns: mbid, name, releases, instruments, recordings
```
[the remark this answers is not in the document]
Comment (miguelgcasado) [reply compmusic/dunya/andalusian.py]: I just copied the style of the others collections. It looks like when there is the description of a dictionary in the return, the line is ended with :: . I'll try to make it consistent.

File: compmusic/dunya/jingju.py [hunk 4]
```diff
@@ -211,3 +213,22 @@ def download_release(release_id, location):
         path = os.path.join(releasedir, name)
         open(path, "wb").write(contents)
 
+def download_score(recordingid, location):
```
Comment (alastair) [inline compmusic/dunya/jingju.py]: this isn't a recording id, right? it's our series:work combination. we should update the argument and documentation to reflect this

**Pull request merged**